Datasets/Cyprus_Pilot/consumer_reports_metadata.json

```diff
@@ -34,7 +34,7 @@
   ],
   "accessRights": "",
   "creator": [
-    "CY Pilot"
+    "CERTH"
```

Comment:
PHOEBE

Reply:
But this is part of the Cypus Pilot, so I should keep it CERTH and just change the person's name, or?

Reply:
Better to put PHOEBE for the moment and then we can change it, just to close it

Reply:
Alright!